.travis.yml

```diff
@@ -29,10 +31,8 @@ matrix:
     - rvm: 2.6.5
       env: DATABASE_ADAPTER=mysql2 DOCKER_IMAGE=huginn/huginn DOCKERFILE=docker/multi-process/Dockerfile
 rvm:
-- 2.3.8
-- 2.4.9
```

Comment:
Great!  I only just decided to drop 2.3 in another PR without looking into this. 😄